openeo-geotrellis/src/test/java/org/openeo/geotrellis/TestOpenEOProcessScriptBuilder.java

```diff
@@ -1232,10 +1231,36 @@ public void testArrayFindNoMatch() {
         ByteArrayTile tile1 = ByteConstantNoDataArrayTile.fill((byte) 5, 4, 4);
 
 
-        Tile emptyResult = transformation.apply(JavaConversions.asScalaBuffer(Arrays.asList(tile0, tile1))).head();
+        Tile emptyResult = transformation.apply(JavaConverters.asScalaBuffer(Arrays.asList(tile0, tile1))).head();
         assertTrue(emptyResult.isNoDataTile());
     }
 
+    @DisplayName("Test array_contains process")
+    @Test
+    public void testArrayContains() {
```

Comment:
is it maybe better to only retain the 'new style' test based on process graph loading?
I would like to try and reduce the volume of test code, for maintainability.

Reply:
removed the obsolete test